perllib/FixMyStreet/App/Controller/Admin/Reports.pm

```diff
@@ -255,7 +255,7 @@ sub edit : Path('/admin/report_edit') : Args(1) {
         $c->detach('edit_display') if $done;
     }
 
-    if ( $c->get_param('resend') && !$c->cobrand->call_hook('disable_resend') ) {
+    if ( $c->get_param('resend') && !$c->cobrand->call_hook('disable_resend_button') ) {
```

Comment:
There's another reference to the `disable_resend` hook in the `edit_category` method in this file, does that need to be changed as well?

https://github.com/mysociety/fixmystreet/blob/bbeafd861b32bf9009d5961053f75e293375ae35/perllib/FixMyStreet/App/Controller/Admin/Reports.pm#L386

Reply:
Yeah, so before this commit there was one `disable_resend` hook that hid the Resend button in the admin, plus also made sure no category change would lead to a report being resent, even if th body or send method changed. After this commit, there are now two hooks, `disable_resend` and `disable_resend_button` - the button one just hides the button in the admin, the `disable_resend` one (not now used by anything at present) still prevents a category change ever resending a report. Thought I might as well leave that in case was useful, but could also remove it as not currently used.

Reply:
Ah I see, that makes sense. Think it's fine to leave it in, seems like something that could well come up in the future again.